### PR TITLE
Hack 25: Detect the user's mobile OS and run a function

### PR DESCRIPTION
**Pull request by brianndiiri**

### CODE REVIEW

File: webflow-hacks/misc-hacks/25/25.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const OTHER_SELECTOR = '[fs-hacks-element="other"]';
+  // get the heading elements
+  const androidHeading = document.querySelector<HTMLHeadingElement>(ANDROID_SELECTOR);
+  const iosHeading = document.querySelector<HTMLHeadingElement>(IOS_SELECTOR);
+  const chromeHeading = document.querySelector<HTMLHeadingElement>(CHROME_SELECTOR);
+  const otherHeading = document.querySelector<HTMLHeadingElement>(OTHER_SELECTOR);
+  if (!androidHeading || !iosHeading || !chromeHeading || !otherHeading) return;
+
+  // If the users window is at or below 991, do everything below
+  if (window.innerWidth > 991) return;
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/25/25.ts]: Not sure about this.
There are Android tablets and iPads that are wider than 991px.
What resources have you researched to decide to add this early return?

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/25/25.ts]: Did not consider tablets, although we can say anything above 991px on webflow should render as desktop?
Should we use the default webflow breakpoints or remove innerWidth altogether?
Screenshot from 2022-06-13 14-44-29

Comment (alexiglesias93) [reply webflow-hacks/misc-hacks/25/25.ts]: The hack is specific for detecting the user's OS.
An Android tablet, even if it's wider than 991px, it's still Android.
I would just remove the innerWidth check

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/25/25.ts]: okay awesome

**Review by alexiglesias93 — APPROVED** [review]

**Pull request merged**